src/runtime_src/driver/xclng/tools/xbutil/xbutil.cpp

```diff
@@ -1479,3 +1493,168 @@ int xcldev::xclP2p(int argc, char *argv[])
 
     return ret;
 }
+
+/*
+ * m2mtest
+ */
+static int m2mtest_bank(xclDeviceHandle handle, uuid_t uuid, int bank_a, int bank_b)
+{
+    const size_t boSize = 3L * 1024 * 1024 * 1024;
+	if (xclOpenContext(handle, uuid, -1, true)) {
+		std::cout << "ERROR: Unable to lockdown xclbin" << std::endl;
+		return -EINVAL;
+	}
+
+    //Allocate boSrc
+    unsigned boSrc = xclAllocBO(handle, boSize, XCL_BO_DEVICE_RAM, bank_a);
+    if (boSrc == NULLBO) {
+        std::cout << "Error allocating Src BO" << std::endl;
+        return -ENOMEM;
+    }
+    void *boSrcPtr = xclMapBO(handle, boSrc, true);
+    if (boSrcPtr == nullptr) {
+        std::cout << "Error mapping Src BO" << std::endl;
+        xclFreeBO(handle, boSrc);
+        return -EINVAL;
+    }
+    memset(boSrcPtr, 'A', boSize);
+	if(xclSyncBO(handle, boSrc, XCL_BO_SYNC_BO_TO_DEVICE, boSize, 0)) {
```

Comment:
strange indent?

Reply:
I am not sure why these strange indents are showing up. They look fine in my editor. I have been trying to fix them but they are hard to fix since I can't really see the difference in my editor

Reply:
Can you enable invisible chars to show up in your editor and see if the indent is what you expected? it could be that the browser is not showing the indent correctly.